lib/rspec/rails/fixture_support.rb

```diff
@@ -50,6 +50,12 @@ def self.proxy_method_warning_if_called_in_before_context_scope(method_name)
             end
           end
 
+          if ::Rails.version.to_f >= 6.1
```

Comment:
I see the error on the CI. `Rails.version` is not good enough?

Reply:
It seems that in this context `Rails` will be resolved to `RSpec::Rails`.

Reply:
nop, it's just 2 tests stubbing the `Rails` module for I don't know what reason https://github.com/rspec/rspec-rails/pull/2215/files#diff-1279af43b24337b9b8a76fc3b3287f9aR199. Build should go green